### ISSUE TITLE
scales-utils.js log attributes (react-vis 1.5.5)

### ISSUE THREAD

**Issue by hqro** [uber/react-vis#452]
Hi,
I have this: `console.log('attributes: ', JSON.stringify(attributes, null, 2));` at line 682  in `./node_modules/react-vis/dist/utils/scales-utils.js` with the last version of react-vis (1.5.5) which log an array each time the crosshair appear when I hover the chart.
```
attributes:  [
  "x",
  "y",
  "radius",
  "angle",
  "color",
  "fill",
  "stroke",
  "opacity",
  "size"
]
```

**Comment by mcnuttandrew** [comment]
Hey @hqro, Can you say a little more about what your issue is? I don't completely understand the error you are highlighting?

**Comment by dignifiedquire** [comment]
I can confirm this issue in `1.5.6` there is a `console.log` statement in the dist build that should not be there, causing the above indicated message to be logged on every render.

```js
// dist/utils/scales-utils.js
...
function getMissingScaleProps(props, data, attributes) {
  var result = {};
  // Make sure that the domain is set pad it if specified
  console.log('attributes: ', JSON.stringify(attributes, null, 2)); // <<<<< this should not be here
  attributes.forEach(function (attr) {
    if (!props[attr + 'Domain']) {
...
```

**Comment by mcnuttandrew** [comment]
Hmm, i'm not seeing it in the code, @CharlieMartell you wanna take a look at this?

**Comment by crunchex** [comment]
Confirmed in to 1.5.5 as well. That bit of code @dignifiedquire pasted is what I'm seeing too.

**Comment by CharlieMartell** [comment]
Hey, @hqro @crunchex , yeah this was my mistake and the reason for the sudden few bumps. I had not realized react-vis did not have a build step in its pre-publish hooks and it was published using a build with a log. This is resolved in 1.5.7. Thanks for bringing this to my attentions!

**Comment by CharlieMartell** [comment]
I made an issue to address the issue and prevent this from happening in the future at: https://github.com/uber/react-vis/issues/456

**Comment by mcnuttandrew** [comment]
I'm gonna close this, as it should be addressed 1.5.7, if anyone see anything differently, feel free to re-open

**Comment by dignifiedquire** [comment]
Looks good in 1.5.7 thank you
